charts/images.yaml

```
@@ -71,33 +71,51 @@ images:
 - name: csi-provisioner
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: "v2.1.2"
+  tag: "v3.2.0"
```

Please use newest release
```suggestion
  tag: "v3.2.1"
```